interface/weno/rxlogmanager.php

```diff
@@ -13,8 +13,10 @@
 use OpenEMR\Common\Acl\AclMain;
 use OpenEMR\Rx\Weno\Container;
 
-//ensure user has proper access
-if (!AclMain::aclCheckCore('admin', 'super')) {
+/*
+ * access control is on Weno side based on the user login
+ */
+if (!AclMain::aclCheckCore('patient', 'med')) {
     echo xlt('ACL Administration Not Authorized');
```

Comment:
I just realized the above message (`ACL Administration Not Authorized`) does not make sense in this script (guessing was copy/pasted from the ACL Administration script :) ).